lib/src/judo_date_time_input.dart

```diff
@@ -118,21 +118,27 @@ class _JudoDateTimeInputState extends State<JudoDateTimeInput> {
             firstDate: widget.firstDate ?? DateTime(1900),
             lastDate: widget.lastDate ?? DateTime(2100),
           );
-          tempTimeOfDay = await showTimePicker(
-              context: context,
-              initialTime: TimeOfDay.fromDateTime(widget.initialDate ?? DateTime.now()),
-              initialEntryMode: TimePickerEntryMode.input,
-              builder: widget.use24HourFormat ? (BuildContext context, Widget child) {
-                return MediaQuery(
-                  data: MediaQuery.of(context).copyWith(alwaysUse24HourFormat: true),
-                  child: child,
-                );
-              }
-                  :
-              null
-          );
 
-          onChangedHandler(DateTime(tempDateTime.year, tempDateTime.month, tempDateTime.day, tempTimeOfDay.hour, tempTimeOfDay.minute));
+          /// If the user cancels the dialog, null is returned.
```

Comment:
What happens if the user tries to "deleted" current date value? If the user switches to text input for date is it allowed to delete value and submit it? What is the returned value if it's allowed?